src/mlpack/methods/reinforcement_learning/replay/prioritized_replay.hpp

```diff
@@ -214,15 +217,15 @@ class PrioritizedReplay
    * @param gradients The model's gradients.
    */
   void Update(arma::mat target,
-              arma::icolvec sampledActions,
+              std::vector<ActionType> sampledActions,
```

Comment:
Can we use ``const std::vector<ActionType>& sampledActions`` here?

Reply:
> Can we use `const std::vector<ActionType>& sampledActions` here?

Yes, I can use `const type &` for `target`, `sampledActions`, `nextActionValues`. The same interface in `random_replay.hpp`.

Reply:
Okay, great.